src/pages/docs/chat/getting-started/kotlin.mdx

```diff
@@ -0,0 +1,1040 @@
+3. Name the project **Chat Example** and place it in the `com.example.chatexample` package
+4. Set the minimum SDK level to API 24 or higher
+5. Select Kotlin as the programming language
+6. Add the Ably dependencies to your app-level `build.gradle.kts` file:
+
+<Code>
+```kotlin
+implementation("com.ably.chat:chat-android:<latest-version>")
+// This package contains extension functions for better Jetpack Compose integration.
+// It's experimental for now (safe to use, but the API may change later). You can always use its code as a reference.
+implementation("com.ably.chat:chat-extensions-compose:<latest-version>")
+```
+</Code>
+
+## Step 1: Setting up Ably <a id="step-1"/>
+
+Replace the contents of your `MainActivity.kt` file with the following code to set up the Ably client:
+
+<Code>
+```kotlin
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.activity.enableEdgeToEdge
+import androidx.compose.foundation.layout.Colum
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import java.util.UUID
+import com.example.chatexample.ui.theme.ChatExampleTheme
+import com.ably.chat.*
+import io.ably.lib.realtime.AblyRealtime
+import io.ably.lib.types.ClientOptions
+
+class MainActivity : ComponentActivity() {
+    private lateinit var realtimeClient: AblyRealtime
+    private lateinit var chatClient: ChatClient
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        realtimeClient = AblyRealtime(
+            ClientOptions().apply {
+                key = "{{API_KEY}}"
+                clientId = "my-first-client"
+            },
+        )
+
+        chatClient = ChatClient(realtimeClient) { logLevel = LogLevel.Info }
+
+        enableEdgeToEdge()
+        setContent {
+            ChatExampleTheme {
+                App(chatClient)
+            }
+        }
+    }
+}
+
+@Composable
+fun App(chatClient: ChatClient) {
+    Scaffold { paddingValues ->
+        Column(
+            modifier = Modifier
+                .fillMaxSize()
+                .padding(paddingValues)
+        ) {
+            Text("Hello Chat App")
+        }
+    }
+}
+```
+</Code>
+
+## Step 2: Connect to Ably <a id="step-2"/>
+
+Clients establish a connection with Ably when they instantiate an SDK. This enables them to send and receive messages in realtime across channels.
+
+In your `MainActivity.kt` file, add the following `ConnectionStatusUi` composable component:
+
+<Code>
+```kotlin
+// This component will display the current connection status
+@Composable
+fun ConnectionStatusUi(connection: Connection) {
+    val connectionStatus = connection.collectAsStatus()
```

Comment:
This might have been just me with Android Studio, when I copy-pasted it couldn't resolve collectAsStatus, but if I deleted the line and re-wrote it with autocomplete, it fixed it...